index.js

```diff
@@ -362,6 +364,14 @@
         }
       });
 
+  //  keys :: StrMap a -> Array String
+  var keys = function(strMap) {
+    if (typeof Object.keys === 'function') return Object.keys(strMap);
```

Comment:
If you really want to test this branch and don't mind restricting your test suite to run in Node only, you could potentially execute the test in a new VM context.

```js
vm.runInNewContext('console.log(Object.keys)', R.dissocPath(['Object', 'keys'], this))
```

That said, how desirable is ES3 support?